doc/man1/flux-env.rst

```diff
@@ -16,7 +16,7 @@ DESCRIPTION
 
 flux-env(1) dumps a list of all environment variables as set by flux if run
 without a command, when run with a command the environment is set and the
-command is run as it would be by the ENV(1) utility.
+command is run as it would be by the :linux:man1:`ENV` utility.
```

Comment:
I think this was meant to be lower case `env(1)`?

Reply:
yeah, i thought that too.  But blindly just went with what was there before.  I'll add a commit to fix this.